wsagent/che-datasource-api/src/main/java/org/eclipse/che/datasource/api/DatasourceModule.java

```diff
@@ -0,0 +1,24 @@
+package org.eclipse.che.datasource.api;
+
+import com.google.inject.AbstractModule;
+import org.eclipse.che.datasource.api.ssl.KeyStoreObject;
+import org.eclipse.che.datasource.api.ssl.SslKeyStoreService;
+import org.eclipse.che.inject.DynaModule;
+import org.eclipse.che.datasource.api.ssl.TrustStoreObject;
+/**
+ * Created by sudaraka on 7/7/17.
+ */
+@DynaModule
+public class DatasourceModule extends AbstractModule{
+    @Override
+    protected void configure() {
+        bind(JdbcConnectionFactory.class);
+        bind(KeyStoreObject.class);
+        bind(TrustStoreObject.class);
+        bind(SslKeyStoreService.class);
```

Comment:
I think we can remove all the ssl stuff. It was meant to have secured connection between the database and che-server. But now every thing would run inside an internal container network. Not sure it is worth it to maintain these ones